server.js

```diff
@@ -437,6 +449,30 @@ app.post('/submitanswer', function(req, res) {
     });
 });
 
+/* check if the submitted MC answer is correct */
+app.post('/submitMCanswer', function(req, res) {
+    questions.checkMCAnswer(req.session.question, req.body.answer,
+                          req.session.user, function(result) {
+        var data = {};
+
+        data.result = result;
+        if (result == 'failed-update') {
+            res.status(500).send(data);
+        } else if (result == 'correct') {
+            data.points = req.session.question.basePoints;
+            /* remove question from questions list, TODO: fetch new one */
+            for (var ind in req.session.questions) {
+                if (req.session.questions[ind].id == req.session.question.id)
+                    break;
+            }
+            req.session.questions.splice(ind, 1);
+            req.session.questionAnswered = true;
+        } else {
+            /* TODO: reload sidebar stats, save in data.html */
```

Comment:
side question: do we have a ticket open for this?

Reply:
nope